docs/validation.md

```diff
@@ -6,15 +6,20 @@ CRI validation testing is currently Alpha and still under quick iterations. We e
 
 ## Install
 
-The test suites can be installed easily via `go get` command:
+The benchmarking tests binary `critest` can be downloaded from [Releasing page](https://github.com/kubernetes-incubator/cri-tools/releases):
 
 ```sh
-go get github.com/kubernetes-incubator/cri-tools/cmd/critest
+wget https://github.com/kubernetes-incubator/cri-tools/releases/download/v1.0.0-beta.0/critest-v1.0.0-beta.0-linux-amd64.tar.gz
```

Comment:
Note that the file does not exist yet, it should be uploaded after beta version released.